schema/attestation.graphqls

```diff
@@ -0,0 +1,39 @@
+scalar JSON
```

Comment:
I think if we're going to have this type then we probably want to use it elsewhere, for VIN VCs?

Reply:
I took this out and made it match string (what is what we use in vin vcs) for now